cdap-client/src/main/java/co/cask/cdap/client/config/ConnectionConfig.java

```diff
@@ -66,10 +67,11 @@ public ConnectionConfig(Id.Namespace namespace, String hostname, int port, boole
     this.hostname = hostname;
     this.port = port;
     this.sslEnabled = sslEnabled;
+    this.username = System.getProperty("user.name");
```

Comment:
Ok then I perhaps misunderstood the requirement. What should be display rather then ``user.name``?